<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLC控制助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>plc地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>105</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>设定电机频率：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>141</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>设定运行状态：</string>
    </property>
    <widget class="QPushButton" name="FW_run">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>正转运行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RW_run">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>反转运行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>661</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>选定分选模式：</string>
    </property>
    <widget class="QPushButton" name="put_list">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>输入序列</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入分选序列：</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>561</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>891</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>电机运转状态：</string>
    </property>
    <widget class="QChartView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>691</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClear">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除曲线</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartAndStop">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>启动定时器</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Freq_set">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionASCII"/>
    <addaction name="action_4"/>
    <addaction name="action1"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_18"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="separator"/>
    <addaction name="action_17"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/actions/stock_undo.png</normaloff>:/actions/stock_undo.png</iconset>
   </property>
   <property name="text">
    <string>读取配置</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/actions/stock_save.png</normaloff>:/actions/stock_save.png</iconset>
   </property>
   <property name="text">
    <string>另存配置</string>
   </property>
  </action>
  <action name="actionASCII">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/actions/text_italic.png</normaloff>:/actions/text_italic.png</iconset>
   </property>
   <property name="text">
    <string>ASCII表</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset>
     <normaloff>actions/add.png</normaloff>actions/add.png</iconset>
   </property>
   <property name="text">
    <string>进制转换</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>快捷键</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>端口设置</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/actions/stock_refresh.png</normaloff>:/actions/stock_refresh.png</iconset>
   </property>
   <property name="text">
    <string>查询更新</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/actions/gtk-cancel.png</normaloff>:/actions/gtk-cancel.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>反馈</string>
   </property>
  </action>
  <action name="action_15">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/actions/stock_bottom.png</normaloff>:/actions/stock_bottom.png</iconset>
   </property>
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="action_17">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/actions/gtk-about.png</normaloff>:/actions/gtk-about.png</iconset>
   </property>
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
  <action name="action_18">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/actions/redo.png</normaloff>:/actions/redo.png</iconset>
   </property>
   <property name="text">
    <string>最小化</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>一个简单的界面</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
